contents/docs/deployment/deploy-source.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Deploying to Source
+sidebar: Docs
+showTitle: true
+---
+
+## Why Source Install
+
+To install from source, you need to build the source code and take care of the dependencies yourself. Likewise, you also need to be aware of all the features of the package so that you can build it accordingly.
+
+Although it may be time consuming and difficult to manage, installing from source has a few advantages:
+
+- Always stay updated, whether it be a security patch or a new feature.
```

Comment:
This is also true for other most of the other ways of deploying.

Reply:
Yep, an oversight; I'll remove that on the next commit.